envelope_test.go

```diff
@@ -869,6 +869,24 @@ func TestUnquotedSpecialCharParamsInMime(t *testing.T) {
 	}
 }
 
+func TestBadAddressHeaderInMime(t *testing.T) {
+	msg := test.OpenTestData("mail", "malformedMultipleAddressHeaderValues.raw")
```

Comment:
```suggestion
	msg := test.OpenTestData("mail", "malformed-multiple-address-header-values.raw")
```
All the other testdata files are kabob-case.  Obv need to rename the actualy file.

Reply:
done